pkg/client/client.go

```diff
@@ -31,7 +31,8 @@ import (
 const (
 	// Value to pass as credentials to the Host Orchestrator service endpoints. Any non-empty value is enough.
 	InjectedCredentials             = "inject"
-	headerNameCOInjectBuildAPICreds = "X-Cutf-Cloud-Orchestrator-Inject-BuildAPI-Creds"
+	HeaderNameCOInjectBuildAPICreds = "X-Cutf-Cloud-Orchestrator-Inject-BuildAPI-Creds"
+	HeaderNameHOBuildAPICreds       = "X-Cutf-Host-Orchestrator-BuildAPI-Creds"
```

Comment:
Use existing `const` https://github.com/google/cloud-android-orchestration/blob/main/pkg/client/host_orchestrator_client.go#L70, need to make it public though.